mesheryctl/internal/cli/root/config/config.go

```diff
@@ -34,12 +34,13 @@ type Token struct {
 
 // Context defines a meshery environment
 type Context struct {
-	Endpoint string   `mapstructure:"endpoint,omitempty"`
-	Token    string   `mapstructure:"token,omitempty"`
-	Platform string   `mapstructure:"platform"`
-	Adapters []string `mapstructure:"adapters,omitempty"`
-	Channel  string   `mapstructure:"channel,omitempty"`
-	Version  string   `mapstructure:"version,omitempty"`
+	Endpoint      string   `mapstructure:"endpoint,omitempty"`
+	Token         string   `mapstructure:"token,omitempty"`
+	TokenLocation string   `mapstructure:"platform" yaml:"token-location,omitempty"`
```

Comment:
@Revolyssup I don't this needs to be added here. Because we do not have this entry in config.yaml

Reply:
The goal is to only print the Token Name and the Location so there's no point in adding this field and eventually this will break a lot of commands as Config.yaml won't be able to Unmarshal into this struct as we don't have this entry originally.